### ISSUE TITLE
Home Assistant 0.92.2, no errors, but cameras show as blue lines

### ISSUE THREAD

**Issue by DrFrankReade** [custom-cards/surveillance-card#14]
In my config, and yes, I seemed to be able to add the custom card without error, my cameras show up as blue lines on a big black background. Running Home Assistant 0.92.2

```
type: custom:surveillance-card
thumb_interval: 15
update_interval: 2
cameras:
   - entity: camera.front_porch
   - entity: camera.side_yard
   - entity: camera.driveway
```

These cameras work fine as picture-glance

I don't have motion detection, so I left those lines out. Even as this card sits there and flounders, the streaming from the cameras works fine in other cards.  

Thoughts?

**Comment by Cjdavidson** [comment]
Yes I have same issue.

**Comment by niemyjski** [comment]
Were you able to get a fix for this? I see it on 0.100 but only very rarely and it's normally when a camera is off.

**Comment by niemyjski** [comment]
The reason you are seeing the blue line is because the camera wasn't found in the hass states. Can you please ensure that your configuration is ok.
